connect/src/indexers/AlchemyClient.ts

```diff
@@ -0,0 +1,82 @@
+import type { Chain, Network } from "@wormhole-foundation/sdk-base";
```

Comment:
This class created from code inside the deleted indexer.ts. Most code is as-is except whats noted below.